types/simple-icons/simple-icons-tests.ts

```diff
@@ -1,9 +1,19 @@
-import simpleIcons from 'simple-icons';
+import simpleIcons = require('simple-icons');
+import abbvie = require('simple-icons/icons/abbvie');
```

Comment:
Perhaps it is better to use `simple-icons/icons/simpleicons` here as we will likely never remove it from the collection. This should prevent us from having to update this test if `abbvie` is removed.

Reply:
Agree, nice suggestion, one sec.